src/function-builder-detectors/netlify-lambda.js

```diff
@@ -26,11 +26,8 @@ module.exports = async function(projectDir) {
     }
   }
 
-  let env = {}
   const envSettings = await getEnvSettings(projectDir)
-  if (envSettings.file) {
-    env = envSettings.vars
-  }
+  const env = envSettings.vars.reduce((env, [key, value]) => ({ ...env, [key]: value }), {})
```

Comment:
It would have been nice to use `Object.fromEntries`....
Also though to also return the `env` object AND the entries, but that's maybe too much?

Reply:
:+1: 
This ponyfill might help: https://github.com/ungap/from-entries/blob/master/index.js

Reply:
After making the change and debugging the code realised this code was completely redundant as we're already setting the env vars on the current process here:
https://github.com/netlify/cli/blob/c5cd7112060151ce31292f424e70656385846cf4/src/commands/dev/index.js#L114
before running this code.
Removed in https://github.com/netlify/cli/pull/1305/commits/baf01c7dac994b7b68f780ec954c4fba3c0325bb